src/templates/finops-hub/schema/schema_ea_normalized.json

```diff
@@ -0,0 +1,723 @@
+        "sink": {
+          "name": "MeterCategory",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "MeterId",
+          "type": "String"
+        },
+        "sink": {
+          "name": "MeterId",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "MeterName",
+          "type": "String"
+        },
+        "sink": {
+          "name": "MeterName",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "MeterRegion",
+          "type": "String"
+        },
+        "sink": {
+          "name": "MeterRegion",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "MeterSubCategory",
+          "type": "String"
+        },
+        "sink": {
+          "name": "MeterSubCategory",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "paygCostInBillingCurrency",
+          "type": "Decimal"
+        },
+        "sink": {
+          "name": "PaygCostInBillingCurrency",
+          "type": "Decimal"
+        }
+      },
+      {
+        "source": {
+          "name": "paygCostInUsd",
+          "type": "Decimal"
+        },
+        "sink": {
+          "name": "PaygCostInUsd",
+          "type": "Decimal"
+        }
+      },
+      {
+        "source": {
+          "name": "PayGPrice",
+          "type": "Decimal"
+        },
+        "sink": {
+          "name": "PayGPrice",
+          "type": "Decimal"
+        }
+      },
+      {
+        "source": {
+          "name": "previousInvoiceId",
+          "type": "String"
+        },
+        "sink": {
+          "name": "PreviousInvoiceId",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "pricingCurrency",
+          "type": "String"
+        },
+        "sink": {
+          "name": "PricingCurrency",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "PricingModel",
+          "type": "String"
+        },
+        "sink": {
+          "name": "PricingModel",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "PartNumber",
+          "type": "String"
+        },
+        "sink": {
+          "name": "ProductId",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "ProductName",
+          "type": "String"
+        },
+        "sink": {
+          "name": "ProductName",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "ProductOrderId",
+          "type": "String"
+        },
+        "sink": {
+          "name": "ProductOrderId",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "OfferId",
+          "type": "String"
+        },
+        "sink": {
+          "name": "ProductOrderName",
```

Comment:
I don't think these values are completely aligned... We should probably split them.

Reply:
Done